src/main/java/org/spongepowered/api/command/args/PatternMatchingCommandElement.java

```diff
@@ -80,7 +80,7 @@ protected Object parseValue(CommandSource source, CommandArgs args) throws Argum
 
     Pattern getFormattedPattern(String input) {
         if (!input.startsWith("^")) { // Anchor matches to the beginning -- this lets us use find()
-            input = "^" + input;
+            input = "^" + Pattern.quote(input); // Quote input to avoid syntax errors caused by characters such as '['
```

Comment:
This is only going to affect some (admittedly, the vast majority of) inputs - if I try to parse an argument input as `^minecraft:stone[`, it'll still fail, because of this if statement.

The better option here would be to treat the input as a literal regardless of what it starts with, quote that, and then _always_ add `^` to the beginning - that way, any argument that uses this element as a base can _also_ start with `^`.